libs/platform/kernel/ebpf_platform_kernel.c

```diff
@@ -31,10 +31,9 @@ ebpf_platform_terminate()
 {}
 
 void*
-ebpf_allocate(size_t size, ebpf_memory_type_t type)
+ebpf_allocate(size_t size)
 {
-    return ExAllocatePool2(
-        type == EBPF_MEMORY_EXECUTE ? POOL_FLAG_NON_PAGED_EXECUTE : POOL_FLAG_NON_PAGED, size, EBPF_POOL_TAG);
+    return ExAllocatePool2(POOL_FLAG_NON_PAGED_EXECUTE, size, EBPF_POOL_TAG);
```

Comment:
Before, almost all callers passed EBPF_MEMORY_NO_EXECUTE which would result in POOL_FLAG_NON_PAGED, so why the change to return executable instead?   The title of the PR talks about removing allocations from executable pool, so I expected this line to say
```suggestion
    return ExAllocatePool2(POOL_FLAG_NON_PAGED, size, EBPF_POOL_TAG);
```

Reply:
Good catch. Yes, I removed the wrong branch.